pandas/_libs/tslibs/period.pyx

```diff
@@ -1217,6 +1217,32 @@ cdef class _Period(object):
 
     @property
     def day(self):
+        """
+        Return the number of days of particular month of a given Period.
+
+        This attribute returns the total number of days of given month on which
+        the particular date occurs.
+        
+        Returns
+        -------
+        Int
+        Number of days till particular date
```

Comment:
This should be indented. Might want to reword based off of what you do with short summary

Reply:
still have confusion  ?

Reply:
Take a look at the contributing guide for the sprint. The very first example should show you how to format the Returns section

https://python-sprints.github.io/pandas/guide/pandas_docstring.html

Reply:
can I write :  
int
   day of that month

Reply:
or day of given month

Reply:
Just write int here - this is simple enough that you don't need a description

For future reference though when you do put a description on the line below the type it needs to be indented (i.e. 4 spaces ahead of the line above)

Reply:
sure